core/wiki/macros/sidebar-resizer.tid

```diff
@@ -0,0 +1,89 @@
+title: $:/core/macros/sidebar-resizer
```

Comment:
Should the macros be tagged, or are they `\import`ed

Reply:
They are imported

Reply:
Maybe some functions could be globals